sources/Application/Instruments/SamplePool.cpp

```diff
@@ -196,6 +196,8 @@ int SamplePool::ImportSample(Path &path) {
   dpath += path.GetName();
   Path dstPath(dpath.c_str());
 
+  Status::Set("Importing %s", path.GetName().c_str());
```

Comment:
Now that this has been moved up here, can probably remove the now redundant identical code on line 163 ?